modules/on-premises/images.yml

```
@@ -0,0 +1,40 @@
+images:
+  - name: Coredns [Fury Kubernetes On-Premises]
+    source: k8s.gcr.io/coredns/coredns
+    tag:
+      - "v1.6.7"
+      - "v1.7.0"
+    destinations:
+      - registry.sighup.io/fury/on-premises/coredns
```

To follow the other images nomenclature I believe this one should be:
```suggestion
      - registry.sighup.io/fury/on-premises/coredns/coredns
```